client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx

```diff
@@ -172,11 +172,12 @@ export const ContentArea = ({
             axis="y"
             renderSpacer={({ ref, style }) => <tr ref={ref} style={style} />}
             initialDelay={1}
+            itemSize={60}
           >
             {line => (
               <RnRFormLine
                 key={line.id}
-                id={line.id}
+                line={line}
```

Comment:
We are already mapping over the lines - might as well use that line, rather than pass the id, to get it separately from zustand store

Reply:
yeah, I went with Andrei's suggestion of using IDs and putting the fetch into the rows. given the refactor back to useQuery the line id is even more pointless than it was